src/site/content/en/blog/stories-component/index.md

```diff
@@ -65,14 +94,41 @@ More specifically, to me that meant:
 }
 ```
 
-{% CompareCaption %}
-Parent with overscroll defines snap behavior
-{% endCompareCaption %}
+Let's breakdown that `grid` value:
 
-{% endCompare %}
+* `1fr` lays out the row, and `auto-flow 100vw` lays out the column
+* [`1fr`](https://alligator.io/css/css-grid-layout-fr-unit/) is
+  technically "1 part of the available space" but in this case you can
+  think of it as filling the height 
+* The placement of `auto-flow` after the `/` is shorthand for 
+  [`grid-auto-flow: column`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow)
+* `100vw` sets the column to the full width of the device
 
-{% Compare 'better', 'child' %}
-```css/3-4
+{% Aside %}
+  Note that the location of the `/` separator relative to `auto-flow` is important.
+  If `auto-flow` came before `/` it would be shorthand for `grid-auto-flow: row`.
+{% endAside %}
+
+On a mobile phone, think of this like the row size being the viewport height
+and each column being the viewport width. Continuing with the Snapchat Stories
+and Instagram Stories example, each column will be a friend's story. We want
+friends stories to continue outside of the viewport so
+we have somewhere to scroll to. Grid will make however many columns it needs to
+layout your HTML for each friend story, creating a perfect scrolling container for us. 
+
+#### Stacking
+
+For each friend we needed their stories in a pagination-ready state.
+In preparation for animation and other fun patterns, I chose a stack. 
+When I say stack, I mean like you're looking down on a
+sandwich, not like you're looking from the side.
```

Comment:
This was previously in an Aside component, but conceptually it's not aside information. Asides are tangential information that are not strictly important for the main idea of the page.